incubator-tools/line_item_comparision/line_item_comparision.ipynb

```diff
@@ -0,0 +1,563 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #137.        def check_page_match(ent1,ent2):
</pre><p>This function is difficult to understand/read due to lots of duplication and multiple layers of nesting. Could be refactored to make it simpler.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/682/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>